assets/src/application/ui/layout/espressoTable/TableDataCell.js

```diff
@@ -24,19 +23,19 @@ const TableDataCell = ({ children, rowNumber, colNumber, htmlId, htmlClass, cssC
 	);
 };
 
-TableDataCell.propTypes = {
-	children: PropTypes.node,
-	rowNumber: PropTypes.number.isRequired,
-	colNumber: PropTypes.number.isRequired,
-	htmlId: PropTypes.string,
-	htmlClass: PropTypes.string,
-	cssClasses: PropTypes.object,
-};
+// TableDataCell.propTypes = {
+// 	children: PropTypes.node,
+// 	rowNumber: PropTypes.number.isRequired,
+// 	colNumber: PropTypes.number.isRequired,
+// 	htmlId: PropTypes.string,
+// 	htmlClass: PropTypes.string,
+// 	cssClasses: PropTypes.object,
+// };
```

Comment:
plz clear out all irrelevant code comments

Reply:
Cleaned up, if anything will check git history for typings.